packages/rocketchat-integrations/lib/rocketchat.js

```diff
@@ -0,0 +1,67 @@
+RocketChat.integrations = {
```

Comment:
Yeah, we don't have anything consistent. This is just what the coffeescript had and was already being referenced by, so I kept it with that.